core/db.go

```
@@ -1,20 +1,27 @@
-package shyftdb
+package core
 
 import (
   	"fmt"
   	"database/sql"
-	"os"
 )
 
 var blockExplorerDb *sql.DB
 
 func InitDB() (*sql.DB, error){
-	var connStr string
-	if "test" == os.Getenv("SHYFT_ENV") {
-		connStr = "user=postgres dbname=shyftdbtest sslmode=disable"
+	var connStr = "user=postgres dbname=shyftdb sslmode=disable"
+	db, err := sql.Open("postgres", connStr)
+	if err != nil {
+		fmt.Println("ERROR OPENING DB, NOT INITIALIZING")
+		fmt.Println(err)
```

If we can't get teh db up should we panic geth instead of logging it?